Makefile

```diff
@@ -38,12 +38,30 @@ lint: lint.sub
 push: push.sub
 test: test.sub
 
+clean:
```

Comment:
Here is a quick example I built to mimic this situation:

`Makefile`:
```
.PHONY: hi

hi:
	echo "hi"

hi:
	echo "hello"
```

Result:
```
$ make hi
Makefile:7: warning: overriding commands for target `hi'
Makefile:4: warning: ignoring old commands for target `hi'
echo "hello"
hello
```

However, if you were to change the targets and use double-colon: `hi::` instead of single-colon, they become additive:

```
.PHONY: hi

hi::
	echo "hi"

hi::
	echo "hello"
```

Result:

```
$ make hi
echo "hi"
hi
echo "hello"
hello
```